Automattic/wp-calypso
# Import in signup: add plans step

This is a 2nd stab at the PR merged in #34231 and then reverted.

#### Changes proposed in this Pull Request

* Adds the plans step to the import flow (after domains)

#### Testing instructions



* Begin a signup from `/start/import`.
* Enter a valid site URL for import.
* If you're a new user, you will be shown the user step.
* You should be able to select a domain.
* You should be taken to the plans step.
* You should be able to checkout.

Note: The eCommerce plan is intentionally omitted, as it makes the target site Atomic, and Atomic sites are currently incompatible with importing in Calypso.
